### ISSUE TITLE
How to convert RetinaNet Model to tensorflow tflite model with metadata

### ISSUE THREAD

**Issue by dheeraj-israni** [keras-team/keras-io#487]
Please can you tell how to convert https://github.com/keras-team/keras-io/blob/master/examples/vision/retinanet.py this model to tensorflow tflite model and what metadata should we add and how.
Thank you
labels: question

**Comment by sayannath** [comment]
Can you show me the code?

**Comment by sachinprasadhs** [comment]
Hi, 

Could you please follow the steps mentioned here https://www.tensorflow.org/lite/models/convert to convert any TensorFlow model.
